core-ui/src/components/Predefined/Details/VirtualServices/HttpRoutes.js

```diff
@@ -0,0 +1,532 @@
+          {rewrite.uri && (
+            <>
+              <dd>{t('virtualservices.http-routes.rewrite.uri')}</dd>
+              <dt>{rewrite.uri}</dt>
+            </>
+          )}
+          {rewrite.authority && (
+            <>
+              <dd>{t('virtualservices.http-routes.rewrite.authority')}</dd>
+              <dt>{rewrite.authority}</dt>
+            </>
+          )}
+        </dl>
+      </LayoutPanel.Body>
+    </LayoutPanel>
+  );
+}
+
+function HttpRetry({ retries }) {
+  const { t } = useTranslation();
+
+  return (
+    <LayoutPanel className="fd-margin--md definition-list">
+      <LayoutPanel.Header>
+        <LayoutPanel.Head
+          title={t('virtualservices.http-routes.retries.title')}
+        />
+      </LayoutPanel.Header>
+      <dl>
+        <dd>{t('virtualservices.http-routes.retries.attempts')}</dd>
+        <dt>{retries.attempts}</dt>
+
+        {retries.perTryTimeout && (
+          <>
+            <dd>{t('virtualservices.http-routes.retries.per-try-timeout')}</dd>
+            <dt>{retries.perTryTimeout}</dt>
+          </>
+        )}
+
+        {retries.retryOn && (
+          <>
+            <dd>{t('virtualservices.http-routes.retries.retry-on')}</dd>
+            <dt>{retries.retryOn}</dt>
+          </>
+        )}
+
+        <dd>
+          {t('virtualservices.http-routes.retries.retryRemoteLocalities')}
+        </dd>
+        <dt>
+          {retries.retryRemoteLocalities
+            ? t('common.labels.yes')
+            : t('common.labels.no')}
+        </dt>
+      </dl>
+    </LayoutPanel>
+  );
+}
+
+function HttpFaultInjectionDelay({ delay }) {
+  const { t } = useTranslation();
+
+  let percent;
+  if (delay.percentage) {
+    percent = delay.percentage.value * 100;
+  } else if (delay.percent) {
+    percent = delay.percent;
+  }
+
+  return (
+    <>
+      <dd>{t('virtualservices.http-routes.fault.delay')}</dd>
+      <dt>
+        {delay.fixedDelay} {percent && `(${percent}%)`}
+      </dt>
+    </>
+  );
+}
+
+function HttpFaultInjectionAbort({ abort }) {
+  const { t } = useTranslation();
+
+  let percent;
+  if (abort.percentage) {
+    percent = abort.percentage.value * 100;
+  }
+
+  return (
+    <>
+      <dd>{t('virtualservices.http-routes.fault.abort')}</dd>
+      <dt>
+        {abort.fixedDelay} {percent && `(${percent}%)`}
+      </dt>
+    </>
+  );
+}
+
+export function StringMatch({ def, ignoreCase }) {
+  const { t } = useTranslation();
+
+  let operator;
+  let value;
+  let tooltip;
+  if (def.exact) {
+    operator = '=';
+    tooltip = t('virtualservices.matches.exact');
+    value = def.exact;
+  } else if (def.prefix) {
+    operator = '^=';
+    value = def.prefix;
+    tooltip = t('virtualservices.matches.prefix');
+  } else if (def.regex) {
+    operator = '~';
+    value = def.regex;
+    tooltip = t('virtualservices.matches.regex');
+  } else {
+    return '-';
+  }
+
+  return (
+    <>
+      <Tooltip className="match-operator" content={tooltip}>
```

Comment:
I had to read the code to realize there's a tooltip here. In my opinion, it should be more obvious for the user. Maybe we could add a question mark icon?